packages/components/src/types.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  OperationVariables,
+  QueryFunctionOptions,
+  QueryResult,
+  BaseMutationOptions,
+  MutationFunction,
+  MutationResult,
+  SubscriptionResult
+} from '@apollo/react-common';
+import { DocumentNode } from 'graphql';
+import { BaseSubscriptionOptions } from '@apollo/react-common/src/types/types';
```

Comment:
This breaks in distribution: https://github.com/apollographql/react-apollo/issues/3117

Reply:
Argh - that's vscode's `typescript.suggest.autoImports` at work 🙁. Fix coming shortly - thanks for pointing this out @patroza!